request_test.go

```diff
@@ -342,6 +356,41 @@ func TestSelectRequestDefaultValues(t *testing.T) {
 	assertBodyEqual(t, refBuf.Bytes(), req)
 }
 
+func TestSelectRequestSpaceByName(t *testing.T) {
+	test_helpers.SkipIfNameUsageUnsupported(t)
```

Comment:
The unit-tests should work any without Tarantool and do not depend on it.

Reply:
Updated.